src/main/java/com/enonic/app/oidcidprovider/PortalRequestBean.java

```diff
@@ -6,7 +6,7 @@
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
-import jakarta.servlet.http.HttpSession;
+import javax.servlet.http.HttpSession;
```

Comment:
I guess we could have started from another branch to support XP 7.15 but this should be fine, too..

This file contains more references to direct oidcidprovider. I think we'll have modify them, too. Hopefully we can later make a generic version of the file that uses application key (available via BeanContext)